docs/changes.rst

```diff
@@ -42,6 +42,10 @@ Released: not yet
 
 **Enhancements:**
 
+* Added a new class `pywbem.NocaseList` that is a case-insensitive but case-preserving
+  list. Objects of this class cannot be created by users, but are returned
```

Comment:
This means that we have to duplicate this in pywbemtools to get the same functionality or use an internal class from pywbem.

Reply:
Should we make it public? And NocaseDict as well?